src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  margin: 0 auto;
```

Comment:
Don't set margins on the BEM block. Better add styles to parent elements to center content.